api/prisma/schema.prisma

```diff
@@ -7,40 +7,204 @@ datasource db {
+  isBackEndCert                Boolean? // Undefined
+  isBanned                     Boolean? // Undefined
+  isCheater                    Boolean? // Undefined
+  isDataAnalysisPyCertV7       Boolean? // Undefined
+  isDataVisCert                Boolean? // Undefined
+  isDonating                   Boolean
+  isFrontEndCert               Boolean? // Undefined
+  isFrontEndLibsCert           Boolean? // Undefined
+  isFullStackCert              Boolean? // Undefined
+  isGithub                     Boolean? // Undefined
+  isHonest                     Boolean?
+  isInfosecCertV7              Boolean? // Undefined
+  isInfosecQaCert              Boolean? // Undefined
+  isJsAlgoDataStructCert       Boolean? // Undefined
+  isLinkedIn                   Boolean? // Undefined
+  isMachineLearningPyCertV7    Boolean? // Undefined
+  isQaCertV7                   Boolean? // Undefined
+  isRelationalDatabaseCertV7   Boolean? // Undefined
+  isRespWebDesignCert          Boolean?
+  isSciCompPyCertV7            Boolean? // Undefined
+  isTwitter                    Boolean? // Undefined
+  isWebsite                    Boolean? // Undefined
+  is2018DataVisCert            Boolean? // Undefined
+  is2018FullStackCert          Boolean? // Undefined
+  keyboardShortcuts            Boolean? // Undefined
+  linkedin                     String? // Null | Undefined
+  location                     String? // Null
+  name                         String? // Null
+  needsModeration              Boolean? // Undefined
+  newEmail                     String? // Null | Undefined
+  partiallyCompletedChallenges Json? // Undefined | PartiallyCompletedChallenge[] TODO: The db only has this as an empty array
+  password                     String? // Undefined
+  picture                      String
+  portfolio                    Portfolio[]
+  profileUI                    ProfileUI? // Undefined
+  progressTimestamps           Json? // ProgressTimestamp[] | Null[] | Int64[] | Double[]
+  rand                         Float? // Undefined
+  savedChallenges              Json? // Undefined | SavedChallenge[]
+  sendQuincyEmail              Boolean
+  sound                        Boolean? // Undefined
+  theme                        String? // Undefined
+  timezone                     String? // Undefined
+  twitter                      String? // Null | Undefined
+  unsubscribeId                String?
+  username                     String
+  usernameDisplay              String? // Undefined
+  verificationToken            String? // Undefined
+  website                      String? // Undefined
+  yearsTopContributor          String[]             @default([]) // Undefined | String[] 
+  a                            String?
+}
+
+// -----------------------------------
+
 model AccessToken {
   id      String   @id @map("_id")
   created DateTime @db.Date
-  /// Multiple data types found: Float: 70.3%, Int: 29.7% out of 118 sampled entries
-  ttl     Json
+  ttl     Float
```

Comment:
This must just have come from running `prisma db pull` again, and the sample taken did not have any of the other types.

Reply:
Okay, I'd like to know what happens if we ask for one, but have the other. I suspect it's bad.

Reply:
I suppose we should audit this as well.

Reply:
Prisma's fairly relaxed about this. It doesn't make a difference if we have Int32 or Double in Mongo, we'll get a number out of Prisma.

The only difference I could find is that if the Prisma schema has Int, it will round the number down to the nearest integer.

So, since ttl should be a integer number of milliseconds, I think we should set this as an Int.
```suggestion
  ttl     Int
```